pyproject.toml

```diff
@@ -64,6 +64,7 @@ mark-parentheses = false
 known-first-party = ["cockpit"]
 
 [tool.vulture]
+min_confidence = 80
```

Comment:
should we set this across other projects as well?

Reply:
Good question, I shall ask @allisonkarlitskaya about it, but in general adding excludes here is quite annoying

Reply:
A lot of the lower-confidence vulture stuff is still high quality...

Reply:
Well not for typed dicts....

```
class ContainerList(TypedDict):
    id: str
    name: str
    image: str
    command: str
    state: list[str] | str
```

Claims `id` / `state` was not used while it is, setting up a vulture suppressions file seems to be too much overhead to me. Alternatively setting this in pyproject.toml with `ignore_names = ["visit_*", "do_*"]` is too much of a :hammer: 

So the only ok solution for me is to bump the confidence level as this was just noise.

Reply:
in fairness, the TypedDict syntax is totally bonkers

see https://github.com/jendrikseipp/vulture/issues/249